src/controllers/answers.js

```diff
@@ -1,8 +1,14 @@
 import Answer from '../models/Answer';
 
 // Return exports list of exportsnswers for exports specific question
-exports.getAnswers = (req, res) => {
-  res.send('NOT IMPLEMENTED: Get a list of Answers');
+exports.getAnswers = async (req, res) => {
+  const results = await Answer.queryGetAnswers(
+    req.params.question_id,
+    req.query.limit,
+    req.query.page,
+    false,
```

Comment:
Are you hard-coding this argument? Will it send 'true' if the request is true?

Reply:
It's not really hard coding because this is an argument required by the query function I'm calling. This value just identifies this function to the called function so the called function can use the proper query text.